### PR TITLE
Update Rule “visual-design-specifications/rule”

### PR DESCRIPTION
**Pull request by hveraus**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/visual-design-specifications/rule.md [hunk 1]
```diff
@@ -19,87 +19,90 @@ created: 2009-02-28T09:45:02.000Z
...
+**Captions:** Each visual should include corresponding s to describes the user’s actions, environment, device, and so on.
 
 ::: good
-![Figure: Good example – Example of wireframes (created in Figma)](figma_wireframe_app_screenshot.png)
+![Figure: Good example – a simple quick hand-drawn storyboards specific scenario with all 3  elements.](basic-storyboard.png)
 :::
 
-Once your low-fidelity design mockup has been approved, you can use it as a reference point for creating a high-fidelity mockup. 
+::: bad
+![Figure: Bad example – UML is virtually impossible to get clients to understand software specifications](bad-uml.jpg)
```
Comment (tiagov8) [inline rules/visual-design-specifications/rule.md:62]: I think this is unrelated and out of context

File: rules/visual-design-specifications/rule.md [hunk 2]
```diff
@@ -19,87 +19,90 @@ created: 2009-02-28T09:45:02.000Z
...
+:::
 
-### High-fidelity Mockups
+### How to create a storyboard
 
-A high-fidelity mockup is a more detailed version of a design that includes visual design elements such as typography, colors, and images. It is created using specialized design tools and takes more time to create than a low-fidelity mockup.
+Creating a storyboard can feel like a daunting task. However, remember they are most valuable as a low-fidelity artifact and don’t need to take a long time to create.
 
-::: greybox
```
Comment (tiagov8) [inline rules/visual-design-specifications/rule.md:56]: 1. change from "greybox" to "info"

File: rules/visual-design-specifications/rule.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 type: rule
-title: Do you determine specifications with a visual design?
+title: Do you conduct specification analysis by storyboarding?
 uri: visual-design-specifications
```
Comment (tiagov8) [inline rules/visual-design-specifications/rule.md:4]: 1. Update the URI (make sure you update the folder name as per Wiki)

File: rules/visual-design-specifications/rule.md [hunk 4]
```diff
@@ -18,88 +18,93 @@ redirects:
...
 
-Clear communication of the message is more important than the medium used to convey that message.
+* [Figma](https://www.figma.com) (Recommended)
+* [Sketch](https://www.sketchapp.com) (Mac Only and for UX designers)
+* [Moqups](https://moqups.com) (HTML5 based App)
+* [UXPin](http://uxpin.com) (more sophisticated, helps you create responsive designs)
 
-::: greybox
+::: good
+![Figure: Good example – Example of a storyboard created in Figma](storyboarding-figma.png)
```
Comment (tiagov8) [inline rules/visual-design-specifications/rule.md:101]: I don't think this is a storyboard example, but a prototype...

> **Storyboarding** can strengthen the user experience elements of your designs. **Wireframe** is a low fidelity layout of the design. **Prototype** captures the intent of a design and simulates multiple states of that design.

**Comment by 2fernandez** [comment]
**Hi Ken,** 
Good work on the rule. Please loop me into conversations about this rule.

1. **Title** - Will you change the title back to "Do you know how to use storyboards?" ?
Note: This is a major requirement from Adam, and I am leaning in this direction.

2. **Introduction** - Can you make a 3-stage diagram showing the evolution of a mockup from, 
Low Fidelity -> High Fidelity or Prototype?

3. **Introduction** - Can you integrate that "storyboards have originated from the cinema industry" no need for a visual, just a mention of this?

4. **Introduction** - Can you illustrate that a "Low Fidelity" mockup is often developed at the "Specification Review" phase?

5. **Low-fidelity Mockups** - Can you first define what "Low-fidelity Mockups" is similar to what is in "High-fidelity Mockups"?

6. **Low-fidelity Mockups** - Can you use an example of an application vs a calculator?

7. **Interactive Fidelity** - I've not heard of this term before. Can you replace it with an Interactive Prototype or Interactive mockup

8. **Tools** - This is a great section; I would recommend leaving out the mention of technology like Figma in areas outside of this section. It will reduce the amount of text in this rule.

File: rules/visual-design-specifications/rule.md (hunk 3 above)
Comment (2fernandez) [reply rules/visual-design-specifications/rule.md:4]: We dont use a storyboard in specification review; we use low-fidelity and high-fidelity mockups.
Storyboarding is more closely associated with a prototype than a mockup.

**Comment by tiagov8** [comment]
> **Hi Ken,** Good work on the rule. Please loop me into conversations about this rule.
> 
> 1. **Introduction** - Can you make a 3-stage diagram showing the evolution of a mockup from,
>    Low Fidelity -> High Fidelity or Prototype?
> 2. **Introduction** - Can you integrate that "storyboards have originated from the cinema industry" no need for a visual, just a mention of this?
> 3. **Introduction** - Can you illustrate that a "Low Fidelity" mockup is often developed at the "Specification Review" phase?
> 4. **Low-fidelity Mockups** - Can you first define what "Low-fidelity Mockups" is similar to what is in "High-fidelity Mockups"?
> 5. **Low-fidelity Mockups** - Can you use an example of an application vs a calculator?
> 6. **Interactive Fidelity** - I've not heard of this term before. Can you replace it with an Interactive Prototype or Interactive mockup
> 7. **Tools** - This is a great section; I would recommend leaving out the mention of technology like Figma in areas outside of this section. It will reduce the amount of text in this rule.

Hey @2fernandez 

Sorry to thumbs down your comment, but I think the major problem of this rule (from the beginning) it that it lacks of accuracy in the terms used and their meanings. 

The things you mentioned are fine, but not for a storyboard rule... 
In this rule they reinforce the mix up of terms... being **not** about storyboards... but mockups, prototypes, wireframes, etc...

Agree?

File: rules/visual-design-specifications/rule.md (hunk 3 above)
Comment (tiagov8) [reply rules/visual-design-specifications/rule.md:4]: Agreed, forget about the specification review. This rule title and URI should talk about storyboards...

E.g. 

title: `Do you know when to use storyboards in software developement?`
uri: `storyboards`

**Comment by 2fernandez** [comment]
> > **Hi Ken,** Good work on the rule. Please loop me into conversations about this rule.
> > 
> > 1. **Introduction** - Can you make a 3-stage diagram showing the evolution of a mockup from,
> >    Low Fidelity -> High Fidelity or Prototype?
> > 2. **Introduction** - Can you integrate that "storyboards have originated from the cinema industry" no need for a visual, just a mention of this?
> > 3. **Introduction** - Can you illustrate that a "Low Fidelity" mockup is often developed at the "Specification Review" phase?
> > 4. **Low-fidelity Mockups** - Can you first define what "Low-fidelity Mockups" is similar to what is in "High-fidelity Mockups"?
> > 5. **Low-fidelity Mockups** - Can you use an example of an application vs a calculator?
> > 6. **Interactive Fidelity** - I've not heard of this term before. Can you replace it with an Interactive Prototype or Interactive mockup
> > 7. **Tools** - This is a great section; I would recommend leaving out the mention of technology like Figma in areas outside of this section. It will reduce the amount of text in this rule.
> 
> Hey @2fernandez
> 
> Sorry to thumbs down your comment, but I think the major problem of this rule (from the beginning) it that it lacks of accuracy in the terms used and their meanings.
> 
> The things you mentioned are fine, but not for a storyboard rule... In this rule they reinforce the mix up of terms... being **not** about storyboards... but mockups, prototypes, wireframes, etc...
> 
> In summary, **storyboards** focus on illustrating user flows, **wireframes** define layout and structure, **prototypes** simulate interactivity, and **mockups** showcase the visual design. These elements are often used in combination to create a comprehensive and user-centered design for software applications.
> 
> Storyboards is possibly the least important one for software developement and the one this rule is about.
> 
> Agree?


All good - I agree.

File: rules/visual-design-specifications/rule.md [hunk 5]
```diff
@@ -19,87 +19,90 @@ created: 2009-02-28T09:45:02.000Z
...
-Many user requirements can be best encapsulated in visual mockups. A design mockup is a visual representation that illustrates the layout, structure, and content of a product or interface before it is developed. A mockup can become a prototype over time but they tend to come earlier in the design process and are static images rather than interactive testing tools.
+Storytelling plays a large role in software design and development. A story captures attention, provides clarity, and inspires teams and stakeholders to take action. 
+
+Storyboards is a good way to visually communicate stories to teams and stakeholders.
+A storyboard is composed of a linear sequence of graphics or illustrations combined to create a visual story.
+
+Used as a tool, storyboards first became popular in motion picture production.
+
+::: good
+![Figure: Good example – Storyboards provided animators to create film before actually building it.](movie-storyboard.jpg)
```
[the remark this answers is not in the document]
Comment (hveraus) [reply rules/visual-design-specifications/rule.md:32]: changed to:

Figure: Good example – Animators from Toy Story movie use storyboards to create a visual storyline before they start creating

File: rules/visual-design-specifications/rule.md (hunk 1 above)
Comment (hveraus) [reply rules/visual-design-specifications/rule.md:62]: How about I change the caption to:

Figure: A UML is good to analyze, design, and document systems but virtually impossible to get clients to understand software specifications

File: rules/visual-design-specifications/rule.md (hunk 3 above)
Comment (hveraus) [reply rules/visual-design-specifications/rule.md:4]: Changed to:

Do you use storyboards in software development?

File: rules/visual-design-specifications/rule.md (hunk 4 above)
Comment (hveraus) [reply rules/visual-design-specifications/rule.md:101]: but I think this type of storyboard are most likely to use in SSW, 

a board of screen mockup linked with arrow to show how the software is going to be used.

File: rules/visual-design-specifications/rule.md (hunk 2 above)
Comment (hveraus) [reply rules/visual-design-specifications/rule.md:56]: Screenshot 2023-09-04 at 15 01 49

@tiagov8 Hi Tiago, the info style seems off, am I using the right one?

**Comment by jaydenalchin** [comment]
Hi @hveraus 

This is excellent stuff, I think splitting the mockup-focused info for designers into a new rule has worked really well!

1. **Capturing requirements** - I think reintroducing more focus on why and how storyboarding can help capture + communicate + clarify complex requirements on a project would make this rule feel more at home under Successful Projects, and make the line between this rule and the new mockups rule even clearer. (I left related feedback on the mockup rule PR)
2. **Name** - Related to the above point, consider renaming to something like "Do you capture complex requirements by storyboarding?"
3. **Intro** - I actually really like introducing storyboarding by addressing its origins in cinema, and the image is a nice visual connection, but I don't think we need to mention "Toy Story" specifically in the figure text. 
4. **Tools** - I agree with comments above about removing Figma, but I would also add Balsamiq https://balsamiq.com/ as a classic and accessible tool anyone can use for simple wireframes/storyboarding

Thanks!

File: rules/visual-design-specifications/rule.md (hunk 1 above)
Comment (tiagov8) [reply rules/visual-design-specifications/rule.md]: I don't like UML being part of the content because I find it unrelated...

File: rules/visual-design-specifications/rule.md (hunk 2 above)
Comment (tiagov8) [reply rules/visual-design-specifications/rule.md:56]: It is correct, but apparently a bug on "info" boxes if you have a list in the second line.

Can you change to:

> Storyboards help to create....
> 
> They can:
> 
> {{ LIST }}

File: rules/visual-design-specifications/rule.md (hunk 4 above)
Comment (tiagov8) [reply rules/visual-design-specifications/rule.md]: The concepts and terms can't be adapted to the way we may do stuff at SSW. These are public rules in the end.

Maybe just say "This prototype made in Figma is generated from a storyboard"

**Comment by tiagov8** [comment]
> Name - Related to the above point, consider renaming to something like _"Do you capture complex requirements by storyboarding?"_

+1 - maybe add _"Do you capture complex **software** requirements by storyboarding?"_

> Intro - I actually really like introducing storyboarding by addressing its origins in cinema, and the image is a nice visual connection, but I don't think we need to mention "Toy Story" specifically in the figure text. 

+1

File: rules/visual-design-specifications/rule.md (hunk 1 above)
Comment (hveraus) [reply rules/visual-design-specifications/rule.md]: It was on Adam's version wo show a bad example trying to use UML to document spec or show to the client.

Personally I also think it is a good "bad example" to show it here.

File: rules/visual-design-specifications/rule.md (hunk 4 above)
Comment (hveraus) [reply rules/visual-design-specifications/rule.md]: changed to:

A prototype created in Figma can also be used effectively as a storyboard

**Comment by tiagov8** [comment]
Hi @hveraus 

As per our conversation on Designers meeting, I am approving this PR since although not 100% by all designers, it's definitely better than what we had before.

**Review by tiagov8 — APPROVED** [review]
